docker/crowdsec/crowdsec/acquis.d/sshd.yml

```
@@ -0,0 +1,4 @@
+filenames:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)